### DISCUSSION TITLE
"if BLOCK" elements are placed in the textarea field.

### DISCUSSION THREAD

**Discussion by upstream-team** [livewire/livewire#7119] [7. Bugs (archived)]
### Livewire version

v3.0.6

### Which PHP version are you using?

PHP 8.2

### Steps To Reproduce

At different periods, the following information appears in the textarea elements: 

`<textarea name="description" class="form-control" placeholder="Description">&lt;!--[if BLOCK]&gt;&lt;![endif]--&gt; &lt;!--[if ENDBLOCK]&gt;&lt;![endif]--&gt;</textarea>
`
I attempted to solve this issue by modifying the configuration file or clearing the cache. The problem periodically disappears, but reappears with another release.

Important! This element is not a part of a Livewire component, and there are no other Livewire components on this page. I am using the package protonemedia/laravel-form-components v3.8.0, and in Blade it looks like this:

`<x-form-textarea name="{{$name}}" placeholder="{{$placeholder}}" default="{{ $value }}" />`

### Please confirm (incomplete submissions will not be addressed)

- [X] I have provided easy and step-by-step instructions to reproduce the bug.
- [X] I have provided code samples as text and NOT images.
- [X] I understand my bug report will be removed if I haven't met the criteria above.

**Comment by naseerr** [comment]
Hello, found a solution in Livewire Github discussions  #7196. Just for the record. If you're using Cache make sure to clear the cache after it :).
